### ISSUE TITLE
Node limit

### ISSUE THREAD

**Issue by ishanpm** [ishanpm/chemlambda-editor#12]
A limiter for the number of nodes would be useful when working with explosive patterns. There should be a limit for the number of nodes on screen, pausing when that threshold is reached. There should also be a cap on the number of moves that can happen in a single frame.

**Comment by mbuliga** [comment]
Good idea, both needed. An extreme example of explosive growth is at https://mbuliga.github.io/feed-the-quine.html if you choose Curry paradox (is from the lambda term related to Curry paradox https://en.wikipedia.org/wiki/Curry%27s_paradox#Lambda_calculus
